qualtran/bloqs/arithmetic.ipynb

```diff
@@ -361,6 +360,118 @@
     "bloq = SquareRealNumber(bitsize=10)\n",
     "show_bloq(bloq)"
    ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "c63ef890",
+   "metadata": {
+    "cq.autogen": "_make_greater_than_constant.md"
+   },
+   "source": [
+    "## `GreaterThanConstant`\n",
+    "Implements $U_a|x\\rangle = U_a|x\\rangle|z\\rangle = |x\\rangle |z ^ (x > a)\\rangle\"\n",
+    "\n",
+    "#### Parameters\n",
+    " - `bitsize`: bitsize of x register.\n",
+    " - `val`: integer to compare x against (a above.) \n",
+    "\n",
+    "#### Registers\n",
+    " - `- x`: Register to compare against val.\n",
+    " - `- result`: Register to hold result of comparison.\n"
```

Comment:
following the PR to appropriately parse the "Registers:" section of a docstring, can you update the registers section of the docstrings to be structured like the Args section. In particular: the docstring should not contain the `-`; that will be added by the system.

Then please re-generate :)

Reply:
Done.